pages/docs/configuration/configuration-mapping.md

```diff
@@ -98,6 +98,26 @@ The interpolation problem might not be well-defined if you map along an axis-sym
 All data mappings are executed during `advance` and not in `readBlockVectorData` etc., cf. the section on  [how to couple your own code](couple-your-code-overview.html).
 {% endnote %}
 
+## Geometric multiscale mapping
+
+Geometric multiscale mapping enables the coupling of dimensionally heterogeneous coupling participants. This can be helpful if some regions of the domain are of less interest and can therefore be treated with simplified, lower dimensional models. One has to differentiate between axial and radial geometric multiscale mapping, i.e. mapping values at an interface boundary surface between two models for the former or coupling along an intersection of the models without such an interface boundary surface. 
```

Comment:
I think that this definition is a bit confusing. In 1D-3D, the main difference is whether we are coupling to one or to multiple points of the 1D mesh. Check again the OFW14 paper: https://mediatum.ub.tum.de/node?id=1515271&change_language=en

We should probably refer to this paper somewhere here anyway.